curvefs/src/client/inode_cache_manager.cpp

```diff
@@ -350,5 +350,57 @@ void InodeCacheManagerImpl::ReleaseCache(uint64_t parentId) {
     iAttrCache_->Release(parentId);
 }
 
+void InodeCacheManagerImpl::FlushInodeLoop() {
+    LOG(INFO) << "flush thread is start.";
+    while (1) {
+        VLOG(9) << "icache size " << iCache_->Size() << " ,maxCacheSize "
+                << maxCacheSize_;
+        if (iCache_->Size() > maxCacheSize_) {
+            DeleteLruInodeCache();
+        } else {
+            sleeper_.wait_for(std::chrono::seconds(flushPeriodSec_));
+            FlushInodeOnce();
+        }
+        if (isStop_.load()) {
+            LOG(INFO) << "flush thread is stop.";
+            return;
+        }
+    }
+}
+
+void InodeCacheManagerImpl::DeleteLruInodeCache() {
+    std::shared_ptr<InodeWrapper> inodeWrapper;
+    uint64_t inodeId;
+    bool ok = iCache_->GetLast(&inodeId, &inodeWrapper);
```

Comment:
At present, the inode cache is synchronously flushed one by one, and the outer loop judgment ensures that it will be flushed all the time, and it will not deviate from the test.